tonic/tonic.go

```diff
@@ -29,7 +29,7 @@ const (
 	RequiredTag   = "required"
 	DefaultTag    = "default"
 	ValidationTag = "validate"
-	ExplodeTag    = "explode"
```

Comment:
Why do you make a change on the tag value ? I think we can keep `explode`, which is the term used in the OpenAPI spec, and it is widely used for this concept.
See https://swagger.io/docs/specification/serialization/, **Path parameters** section.

Reply:
This is open for debate, I just felt the explode semantics are really unclear.
It was extremely confusing again getting back into it for this PR even though I was familiar with the various permutations, so I cant imagine what it must be like fresh off.